proto/rpc/v1/tx.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package rpc.v1;
+
+import "rpc/v1/transaction.proto";
+import "rpc/v1/meta.proto";
+
+message TxRequest {
+    // the hash of the transaction
+    bytes hash = 1;
+
+    // if true, return data in binary format
+    // TODO: does this make sense for protobuf?
```

Comment:
@keefertaylor what do you think about this? Is there any benefit to returning this data in binary format? protobuf is already a binary format over the wire and is heavily optimized, so I don't think it makes sense to provide a binary option for efficiency reasons. But maybe there are other reasons a client would want the data in binary form.

Reply:
As far as I can tell I don't think there is. 

It would be easiest to leave this off for now, and if we find a use case we really must have later we can always add it back in (conversely, it's harder to remove stuff).

Reply:
A simple use case would be that this is the bulk of the data one needs to verify hashes. If data only arrives in some deserialized format, one first needs tom implement a serializer to verify incoming data.

Reply:
I kept this and removed the comment